### PR TITLE
[Natalia Falinska] Develop network layer service

### PR DESCRIPTION
**Pull request by NFalinska**

### CODE REVIEW

File: src/app/data.service.ts [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+})
+export class DataService {
+  private baseURL: String = environment.baseURL;
+  private headers: HttpHeaders = new HttpHeaders({
+    'Content-Type':  'application/json'
+  });
+
+  constructor(private http: HttpClient) { }
+
+  private handleError(error: HttpErrorResponse) {
```
Comment (dverbovyi) [inline src/app/data.service.ts]: public properties and method should be first, than private ones

Please refactor this according to style guide https://angular.io/guide/styleguide#member-sequence

File: src/app/data.service.spec.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { TestBed } from '@angular/core/testing';
+
+import { DataService } from './data.service';
+
+describe('DataService', () => {
+  beforeEach(() => TestBed.configureTestingModule({}));
+
+  it('should be created', () => {
```
Comment (dverbovyi) [inline src/app/data.service.spec.ts]: add more test for all your public methods

File: src/environments/environment.prod.ts [hunk 3]
```diff
@@ -1,3 +1,5 @@
 export const environment = {
-  production: true
+  production: true,
+  baseUrl: 'http://localhost:4200'
```
Comment (AlexLarusso) [inline src/environments/environment.prod.ts]: move port to separate property

File: src/environments/environment.ts [hunk 4]
```diff
@@ -3,7 +3,8 @@
 // The list of file replacements can be found in `angular.json`.
 
 export const environment = {
-  production: false
+  production: false,
+  baseURL: 'http://localhost:4200'
```
Comment (AlexLarusso) [inline src/environments/environment.ts]: same

File: package.json [hunk 5]
```diff
@@ -17,43 +17,44 @@
...
   "dependencies": {
-    "@angular/animations": "~7.1.0",
-    "@angular/common": "~7.1.0",
-    "@angular/compiler": "~7.1.0",
-    "@angular/core": "~7.1.0",
-    "@angular/forms": "~7.1.0",
-    "@angular/platform-browser": "~7.1.0",
-    "@angular/platform-browser-dynamic": "~7.1.0",
-    "@angular/router": "~7.1.0",
+    "@angular/animations": "^7.1.4",
```
Comment (AlexLarusso) [inline package.json]: Do you really need animations for dataService?
Remove useless dependencies.

**Pull request merged**